[CI:DOCS] Add play kube support docs (#14318)
containers/podman: docs/play_kube_support.md

# Podman Play Kube Support

> This document outlines the kube yaml fields that we currently support with the **podman play kube** command.

Reviewer: ```suggestion
This document outlines the kube yaml fields that are currently support by the **podman play kube** command.
```

Reviewer 2: Nice eye Valentin, no pronouns in man pages please!

> Note: **N/A** means that we will not support that option because it doesn't make sense in the podman world.

Reviewer: ```suggestion
Note: **N/A** means that the option cannot be supported in a single-node Podman environment.
```
